test-common/misc-utils.c

```diff
@@ -22,119 +22,18 @@
 
 #include "misc-utils.h"
 
+#include "libeos-updater-util/util.h"
+
 #include <errno.h>
 
 #include <unistd.h>
 
-static gboolean
-rm_file_ignore_noent (GFile         *file,
```

Comment:
Dropping this copy of the `rm_rf*()` functions should be done in the previous commit which moves them into `libeos-updater-util`, I think?

Reply:
I've moved it into a commit after when we link the tests to libeos-updater-util